edge/pkg/edged/status/status_manager.go

```diff
@@ -3,17 +3,17 @@ package status
 import (
 	"time"
 
-	"github.com/kubeedge/kubeedge/pkg/edged/podmanager"
+	"github.com/kubeedge/kubeedge/edge/pkg/edged/podmanager"
```

Comment:
please move this line together with the github.com/kubeedge/kubedge/* import section

Reply:
fixed